components/cmake/modules/FindCsmShare.cmake

```diff
@@ -29,7 +29,7 @@ endif()
 
 # Create the interface library, and set target properties
 add_library (csm_share INTERFACE)
-target_link_libraries (csm_share INTERFACE ${CSM_SHARE_LIB};mct;${PIOLIBS})
+target_link_libraries (csm_share INTERFACE ${CSM_SHARE_LIB};mct;spio)
```

Comment:
Should we maybe to the same thing we did for MCT also for spio? That is, adding
```
find_package(PIO)
```
?

Reply:
Yep, did exactly that.

Reply:
I meant adding it inside the `FindCsmShare.cmake` module.

Reply:
Oh, I see. Yes let me do that.